frontend/src/metabase/browse/components/BrowseDatabases.tsx

```diff
@@ -30,34 +36,76 @@ export const BrowseDatabases = ({
     return <LoadingAndErrorWrapper loading />;
   }
 
-  return databases?.length ? (
-    <DatabaseGrid data-testid="database-browser">
-      {databases.map(database => (
-        <div key={database.id}>
-          <DatabaseCardLink to={Urls.browseDatabase(database)}>
-            <DatabaseCard>
-              <Icon
-                name="database"
-                color={color("accent2")}
-                className={CS.mb3}
-                size={32}
-              />
-              <Title order={2} size="1rem" lh="1rem" color="inherit">
-                {database.name}
-              </Title>
-            </DatabaseCard>
-          </DatabaseCardLink>
-        </div>
-      ))}
-    </DatabaseGrid>
-  ) : (
-    <CenteredEmptyState
-      title={<Box mb=".5rem">{t`No databases here yet`}</Box>}
-      illustrationElement={
-        <Box mb=".5rem">
-          <img src={NoResults} />
-        </Box>
-      }
-    />
+  if (!databases?.length) {
+    return (
+      <CenteredEmptyState
+        title={<Box mb=".5rem">{t`No databases here yet`}</Box>}
+        illustrationElement={
+          <Box mb=".5rem">
+            <img src={NoResults} />
+          </Box>
+        }
+      />
+    );
+  }
+
+  return (
+    <BrowseContainer>
+      <BrowseHeader>
+        <BrowseSection>
+          <Flex w="100%" direction="row" justify="space-between" align="center">
+            <Title order={1} color="text-dark">
+              <Group spacing="sm">
+                <Icon color={color("brand")} name="database" />
+                {t`Databases`}
+              </Group>
+            </Title>
+            <LearnAboutDataLink />
+          </Flex>
+        </BrowseSection>
```

Comment:
this header also looks big

![Screen Shot 2024-05-07 at 3 20 45 PM](https://github.com/metabase/metabase/assets/30528226/ee7c3253-2007-42f6-a5d3-bc8e00f0a6ce)

Reply:
This matches the Browse models page but I've [pinged](https://metaboat.slack.com/archives/C06REDHM6BV/p1715165222186739) Dawei to double check